### ISSUE TITLE
Hitching "Routes" ("trails")

### ISSUE THREAD

**Issue by dcht00** [Hitchwiki/hitchwiki#80]
I'd like to introduce more of routes / trails to the site.

(Thinking though a graph) Hitchwiki, hospitality sites, etc seem to only have Nodes, not Edges. Travel is more a "directed graph" - A to B, with stops.

Currently:
- http://hitchwiki.org/en/Routes : a category page with very few routes
- http://hitchwiki.org/en/Amsterdam#East_towards_Amersfoort.2C_Hengelo.2C_Osnabr.C3.BCck.2C_Hamburg.2C_Bremen.2C_Berlin

---

I noticed there are standard trails I'm hitching repeatedely. For reference and example, I've defined a few basic routes to traverse the European continent here: http://maps.totalism.org (under Routes/Trails)

More ideas:
- people would do more long-distance hitching if they would be able to see dots on the way from A to B, and have a way to imagine the full trip, not just a way to escape A :)
- there's a history of standard trails that is kind of lost; this would be a great format to bring it back
- co-hitching: this would work great for deciding to do a route and find others to tour with!
- this would also connect well with Trustroots.
- maybe systematic analysis would show where to focus efforts around these trails (with nodes/stops & stuff to do around every ~300km).
- interesting: I noticed I enjoy travelling with an idea what a trail is in general, but making variations ("sub branches")

---

Tasks & Outcomes:

My suggestions would be to:

1) add "Routes" to a visible spot on page, like "How to hitchhike"

2) create a smart template for including Routes on pages (I can do this)

Basically like here: http://hitchwiki.org/en/Amsterdam#East_towards_Amersfoort.2C_Hengelo.2C_Osnabr.C3.BCck.2C_Hamburg.2C_Bremen.2C_Berlin

3) template 2.0 would be:
    \* a way to describe a route
    \* possibly a script auto-embedding it to place pages

4) A default "Maps", or atleast one in Routes, could show Europe (World) with basic trails covering capitals + prominent cities.

**Comment by guaka** [comment]
ping @simison
